frontend/src/components/WordTree/styles/WordTree.css

```diff
@@ -17,14 +17,63 @@
     margin: 25px 0;
 }
 
-.rd3t-link{
+.rd3t-link {
     stroke: var(--branchcolor) !important;
 }
 
-.zoomGroup .MuiButtonGroup-grouped:not(:last-of-type){
+.zoomGroup .MuiButtonGroup-grouped:not(:last-of-type) {
     border-color: var(--zoomcolor) !important;
 }
 
-.zoomGroup .css-d4hd0p-MuiButtonBase-root-MuiButton-root{
+.zoomGroup .css-d4hd0p-MuiButtonBase-root-MuiButton-root {
     color: var(--zoomcolor) !important;
-}
\ No newline at end of file
+}
+
+.zoomGroup {
+  position: absolute;
+  right: 20px;
+  top: 20px;
+}
+
+.drawer {
+  width: 250px;
+}
+
+.collapsible-list-item {
+    padding-left: 20px;
+  }
+  
+  .collapsible-list-item-level-1 {
```

Comment:
try to be consistent with the file indentation. again, a linter would detect this.